### PR TITLE
Fix CircleCI "g++ can't be a slave of gcc" error.

### PR DESCRIPTION
**Pull request by scottschurr**
Hopefully this will get our CircleCI integration back on the air.

Reviewers: @seelabs, @nbougalis.  Interested observer: @wilsonianb

### CODE REVIEW

**Comment by vinniefalco** [comment]
Those gcc slaves...

File: circle.yml [hunk 1]
```diff
@@ -10,8 +10,10 @@ dependencies:
     - sudo apt-get update -qq
     - sudo apt-get purge -qq libboost1.48-dev
     - sudo apt-get install -qq libboost1.57-all-dev
-    - sudo apt-get install -qq clang-3.6 gcc-5 libobjc-5-dev libgcc-5-dev libstdc++-5-dev libclang1-3.6 libgcc1 libgomp1 libstdc++6 scons protobuf-compiler libprotobuf-dev libssl-dev exuberant-ctags
-    - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-5 99 --slave /usr/bin/g++ g++ /usr/bin/g++-5
+    - sudo apt-get install -qq clang-3.6 gcc-5 g++-5 libobjc-5-dev libgcc-5-dev libstdc++-5-dev libclang1-3.6 libgcc1 libgomp1 libstdc++6 scons protobuf-compiler libprotobuf-dev libssl-dev exuberant-ctags
+    - lsb_release -si -sr
```
Comment (seelabs) [inline circle.yml]: I'd use `lsb_release -d` for this.

File: circle.yml [hunk 2]
```diff
@@ -10,8 +10,10 @@ dependencies:
...
+    - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-5 99
+    - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-5 99
```
Comment (seelabs) [inline circle.yml]: I think the slave error was a symptom of the missing `apt-get install g++-5`. I'd prefer to keep g++ a slave of gcc, as that's how we'd normally do upgrade the compiler.

It's only a nit though. It's not like we're going to install gcc-5 and then use `update-alternatives` to revert back to gcc 4.

**Comment by seelabs** [comment]
:+1: I pointed out two nits, I'd pass it without any changes as well.

**Comment by scottschurr** [comment]
I just tried the `update-alternatives --slave` formulation after `install g++-5`.  The slave formulation is still not working, even with the `install g++-5`.

**Comment by seelabs** [comment]
@scottschurr Oh well, thanks for trying.:+1: assuming the checks pass. (I also saw you changed the `lsb_release` param).

**Comment by scottschurr** [comment]
Merged as 1e1aa761398e4509a52d24d009275b8c4d82d970.

**Pull request closed without merging**